scripts/build-bmo-push-data.pl

```diff
@@ -199,7 +210,13 @@ sub _get {
   }
 
   my $response = $ua->request($request);
-  if ($response->code !~ /^2/) {
+  if ($response->code =~ /^2/) {
+    return decode_json($response->decoded_content);
+  }
+  elsif ($response->code =~ /^4/) {
+    return {};
+  }
```

Comment:
Is this just used for Github integration, or will this caught case have unwanted behaviour changes elsewhere?

Reply:
This only affects creating text file artifacts in CircleCI that help me with posting announcements to various locations such as the wiki, blog and firefox-tools discourse. It doesn't affect anything outside of that.